python/cuml/decomposition/tsvd.pyx

```diff
@@ -95,6 +97,11 @@ cdef extern from "cuml/decomposition/tsvd.hpp" namespace "ML":
                             const paramsTSVD &prms) except +
 
 
+class Solver(IntEnum):
```

Comment:
Was this the trick to expose the enum class through cython?

Reply:
Yeah.. it's weird. The cython removes the scope of the `enum class` and forces it to be `enum`. We scope it back with this. Added advantage of being able to pass the new scoped enum around as a python object